lib/compute/jenkins-main-node.ts

```diff
@@ -191,6 +198,8 @@ export class JenkinsMainNode {
       InitPackage.yum('java-1.8.0-openjdk-devel'),
       InitPackage.yum('openssl'),
       InitPackage.yum('mod_ssl'),
+      // eslint-disable-next-line max-len
+      InitCommand.shellCommand('sudo wget -nv https://github.com/mikefarah/yq/releases/download/v4.22.1/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'),
```

Comment:
That is this repository used for?

Reply:
You cannot add aws secrets values in typescript. I am getting values and replacing them in already loaded yaml on the server

Reply:
Can we rather extract the values in EC2, return them and use them in code?

Reply:
Nope, that will over complicate this unnecessarily. Plus from security point of view it is better that the secrets stay on server than written to local file.

Reply:
The secret would still be written on the file on the server, just like now.

Reply:
discussed offline